ogc/open-street-map-layer/src/main/java/com/esri/samples/open_street_map_layer/OpenStreetMapLayerSample.java

```diff
@@ -44,13 +46,20 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create map with an OpenStreetMap basemap
-      ArcGISMap map = new ArcGISMap(Basemap.Type.OPEN_STREET_MAP, 34.056295, -117.195800, 10);
-      
-      // set the map to a map view
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with an OpenStreetMap basemap style
```

Comment:
Maybe it's more correct to say `the` basemap style, since it's an enum?
```suggestion
      // create a map with the OpenStreetMap basemap style
```